ray-operator/controllers/ray/utils/util.go

```diff
@@ -601,3 +601,10 @@ type ClientProvider interface {
 	GetDashboardClient(mgr manager.Manager) func() RayDashboardClientInterface
 	GetHttpProxyClient(mgr manager.Manager) func() RayHttpProxyClientInterface
 }
+
+func ManagedByExternalController(controllerName *string) *string {
```

Comment:
Do we still need this? Seems like in rayjob_controller.go we have a local version managedByExternalController?

Reply:
Oh...
I will use this one instead as it's going to be universal for both ray job and ray cluster